### PR TITLE
[BTS-167]: SnackBar Component

### PR DESCRIPTION
**Pull request by rapahaeru**
http://vm-tmb-jirasoft01.servers/browse/BTS-167

### CODE REVIEW

File: components/SnackBar/SnackBar.jsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+  componentDidMount() {
+    const { body } = document;
+    body.appendChild(this.snackBarSection);
+
+    this.objectFocus(document.querySelector('.action-button'));
+
+    const { onClose, secondsToClose } = this.props;
+
+    setTimeout(() => {
```
Comment (allyssonsantos) [inline components/SnackBar/SnackBar.jsx]: I think you must save this `setTimeout` to a class property to "clear" this on componentWillUnmount();

If you open a snackbar and count to 3, click on close icon, open another snackbar and wait, instead of closing in 6 seconds the new one will close after 3 seconds.

File: components/SnackBar/SnackBar.jsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  handleKeyPress = event => {
+    const { onClose } = this.props;
+    if (event.key === 'Enter') onClose();
+  };
+
+  objectFocus = element => {
+    element.focus();
+  };
+
+  closeOnTime = (secondsToClose, callback) => {
```
Comment (allyssonsantos) [inline components/SnackBar/SnackBar.jsx]: this setTimeout is equal the one that is on line 98, why did you need this?

File: components/SnackBar/SnackBar.jsx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      text,
+      onClose,
+      closeButtonAriaLabel,
+      actionTrigger,
+      ...rest
+    } = this.props;
+    return ReactDOM.createPortal(
+      <DialogBlock>
+        <Row>
+          <Col xsmall-offset={1} xsmall={10}>
```
Comment (allyssonsantos) [inline components/SnackBar/SnackBar.jsx]: This values are wrong, the xsmall have only 4 columns, small 6 columns, medium, large and xlarge have 12 columns.

If you test on a small resolution you will see that the snackbar is at the right corner of the viewport.

image

File: components/SnackBar/SnackBar.jsx (hunk 2 above)
Comment (rapahaeru) [reply components/SnackBar/SnackBar.jsx]: Oh! I Forgot this line, sorry
